packages/calcite-components/src/components/accordion-item/accordion-item.tsx

```diff
@@ -283,7 +291,13 @@ export class AccordionItem extends LitElement {
           [`icon-type--${this.iconType}`]: true,
         }}
       >
-        <div class={{ [CSS.header]: true, [CSS_UTILITY.rtl]: dir === "rtl" }}>
+        <div
+          class={{
+            [CSS.header]: true,
+            [CSS_UTILITY.rtl]: dir === "rtl",
+            [`header--${this.appearance}`]: true,
```

Comment:
nitpick: it would be nice if these classes were in the CSS object so they could be used in an e2e test if needed.